src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalSum = days * 40;
+
+  if (days < 3) {
+    return totalSum;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
`days` will always be >= 3 at this point
```suggestion
  if (days < 7) {
```

Reply:
You are right, thanks for reviewing.